src/views/List/List.jsx

```diff
@@ -17,11 +22,29 @@ export default function List({ token }) {
   }, []);
 
   return (
-    <ul>
-      {data.map((listItem, index) => {
-        const { name } = listItem;
-        return <li key={index}>{name}</li>;
-      })}
-    </ul>
+    <section>
```

Comment:
<section> semantic html we love to see it 👏

it's huge for the accessibility of your websites and I've recently realized as a dev how much nicer and easier it is to read html with <section><article> and other semantic containers versus <div>s everywhere